core/common/src/rpc/IModelTileRpcInterface.ts

```diff
@@ -45,6 +48,7 @@ export abstract class IModelTileRpcInterface extends RpcInterface {
   }
 
   /** @internal */
+  @RpcOperation.allowResponseCaching(RpcResponseCacheControl.Immutable)
```

Comment:
Same comment here about interface version and backwards compatibility.

Reply:
bumped minor version